imap_processing/tests/codice/test_decompress.py

```diff
@@ -39,3 +38,10 @@ def test_decompress(compressed_value: int, algorithm: IntEnum, expected_result:
 
     decompressed_value = decompress(compressed_value, algorithm)
     assert decompressed_value == expected_result
+
+
+def test_decompress_raises():
+    """Tests that the ``decompress`` function raises with an unknown algorithm"""
+
+    with pytest.raises(ValueError, match="some_unsupported_algorithm is not"):
```

Comment:
Wasn't sure if this was here on purpose

```suggestion
    with pytest.raises(ValueError, match="some_unsupported_algorithm"):
```

Reply:
I did actually do it on purpose but it does read funny when it abruptly ends like that, so good suggestion!